front/app/containers/Admin/formBuilder/components/FormBuilderSettings/PageSettings/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+// components
+import InputMultilocWithLocaleSwitcher from 'components/HookForm/InputMultilocWithLocaleSwitcher';
+import { SectionField } from 'components/admin/Section';
+import QuillMultilocWithLocaleSwitcher from 'components/HookForm/QuillMultilocWithLocaleSwitcher';
+
+// types
+import { IFlatCustomFieldWithIndex } from 'services/formCustomFields';
+
+const PageSettings = (field: IFlatCustomFieldWithIndex) => (
+  <>
+    <SectionField>
+      <InputMultilocWithLocaleSwitcher
+        id="e2e-title-multiloc"
```

Comment:
Minor: Let's rename this `e2e-page-title-multiloc` since we have `e2e-title-multiloc` this somewhere else. It will make it easy to track